core/src/main/java/io/confluent/kafka/schemaregistry/metrics/MetricsContainer.java

```diff
@@ -146,6 +148,9 @@ public MetricsContainer(SchemaRegistryConfig config, String kafkaClusterId) {
 
     this.protobufSchemasDeleted = createMetric(METRIC_NAME_PB_SCHEMAS_DELETED,
             "Number of deleted Protobuf schemas", new CumulativeCount());
+
+    this.leaderInitializationLatency = createMetric(METRIC_LEADER_INITIALIZATION_LATENCY,
```

Comment:
Does it makes sense to add a metric for number of leader changes. This would help observer this before and after you changes.

Reply:
we can deduce this from the log messages. i don't feel a metric is needed for this. a more useful metric is to track the time it takes to initialize the leader as the SR cluster cannot accept any writes until this is done. i've introduced this metric in this PR. DD also charges per metric IIUC.